src/backend/renderer/gles/mod.rs

```diff
@@ -2570,6 +2569,7 @@ impl<'frame> GlesFrame<'frame> {
 
         // as a last option we force finish, this is unlikely to happen
         unsafe {
+            self.renderer.gl.Flush();
```

Comment:
I don't think flush buys anything here.

Reply:
Yeah, right, glFinish alone should be enough.

Reply:
removed the unnecessary flush